src/DynamoSandbox/DynamoSandbox.csproj

```diff
@@ -18,11 +21,12 @@
   <PropertyGroup>
     <ApplicationIcon>logo_square_32x32.ico</ApplicationIcon>
   </PropertyGroup>
-
   <ItemGroup>
     <Reference Include="Microsoft.Practices.Prism">
       <HintPath>..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
```

Comment:
so does this actually still function without being updated?

Reply:
it seems to. Unless we get a version of it specifically built for net6. we won't have a lot of guarantees that all APIs are compatible.